.github/workflows/self-scheduled-caller.yml

```diff
@@ -7,7 +7,8 @@ on:
     - cron: "17 2 * * *"
   push:
     branches:
-      - run_scheduled_ci*
+      # - run_scheduled_ci*
+      - fix-ci-quantization
```

Comment:
```suggestion
      - run_scheduled_ci*
```

Reply:
yes ! just testing to see if everything works as expected this time ;)